app/src/main/res/layout/activity_widget_config.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_gravity="center_horizontal"
+    android:background="#ff262C31"
+    android:gravity="center_vertical"
+    android:orientation="vertical"
+    android:padding="16dp"
+    tools:context="de.danoeh.antennapod.activity.WidgetConfigActivity">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="16dp"
+        android:layout_marginTop="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginBottom="16dp"
+        android:gravity="center_horizontal|center_vertical"
+        android:orientation="horizontal">
+
+        <TextView
+            android:id="@+id/textView"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/widget_opacity"
+            android:textColor="#FFFFFF"
+            android:textSize="18sp"
+            android:textStyle="bold" />
+
+        <TextView
+            android:id="@+id/widget_opacity_textView"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:gravity="end"
+            android:text="100%"
+            android:textColor="#FFFFFF"
+            android:textSize="18sp" />
+    </LinearLayout>
+
+    <FrameLayout
```

Comment:
Great to see that you added the widget. Just a feeling but I think the widget preview should be above the settings. Have you experimented with moving the preview background, so that this does not have padding on the left/right/top? That way, it would look a bit like the about screen.

Reply:
![widget](https://user-images.githubusercontent.com/56698158/67337750-ecfc5400-f527-11e9-9816-90949de8274a.png)

Something like this?

Reply:
Yeah, I like it like this. Feel free to disagree, though.

I just noticed that you manually changed the activity background and text color. If you remove those colors and add `setTheme(UserPreferences.getTheme());` to the first line of the activity's onCreate method, it will respect the theme setting.

Reply:
I like it more like this too when it matches with app.